src/components/inputField/selectElement/SelectElement.tsx

```diff
@@ -0,0 +1,111 @@
+import { useEffect, useRef, useState } from "react";
+import CarretDownIcon from "../../../assets/images/icon-caret-down.svg?react";
+import "../inputField.scss";
+
+interface selectProps {
+    dropdownOpened: boolean;
+    selectDecorator:string | undefined;
+}
+const colorTags = [
+    "green", "yellow", "cyan", "navy", "red", "purple", "turquoise", "brown", "magenta", "blue", "navy-grey", "army-green", "gold", "orange" 
+]
+const transactionCategories = [
+    "All Transactions", "Entertainment", "Bills", "Groceries", "Dining Out", "Transportation", "Personal Care", "Education", "Lifestyle", "Shopping", "General"
+]
+
+export default function SelectElement({dropdownOpened, selectDecorator}:selectProps){
```

Comment:
This component needs to establish what's the data structure that it receives to render through props and render that. Now you've hardcoded two data sets. What happens if I have 4 new data sets that need to be presented to the user in a selector?